crypto/merkle/simple_tree.go

```diff
@@ -32,6 +32,7 @@ func SimpleHashFromMap(m map[string][]byte) []byte {
 	return sm.Hash()
 }
 
+// getSplitPoint returns the largest power of 2 less than length
```

Comment:
Not sure if we should refer to the spec here too? https://tools.ietf.org/html/rfc6962#section-2.1.2